src/views/studio/MyVideosView/MyVideosView.tsx

```diff
@@ -367,3 +388,32 @@ const getPublicness = (currentTabName: typeof TABS[number]) => {
       return undefined
   }
 }
+
+type NewVideoTileProps = {
+  loading?: boolean
+}
+
+const NewVideoTile: React.FC<NewVideoTileProps> = ({ loading }) => {
+  return (
+    <SwitchTransition>
+      <CSSTransition
+        key={loading ? 'cover-placeholder' : 'cover'}
+        timeout={parseInt(transitions.timings.sharp)}
+        classNames={transitions.names.fade}
+      >
+        <NewVideoTileWrapper>
+          {loading ? (
+            <NewVideoTileSkeleton />
+          ) : (
+            <NewVideoTileLink to={absoluteRoutes.studio.editVideo()}>
+              <TextAndIconWrapper>
+                <StyledIcon />
+                <StyledText variant="body2">Upload new video</StyledText>
+              </TextAndIconWrapper>
+            </NewVideoTileLink>
+          )}
+        </NewVideoTileWrapper>
+      </CSSTransition>
+    </SwitchTransition>
+  )
+}
```

Comment:
Let's move this to a separate file, along with its styles. It can be here or in one of the `components` folders